code/modules/mob/living/carbon/xenomorph/hive_datum.dm

```diff
@@ -500,7 +500,7 @@ to_chat will check for valid clients itself already so no need to double check f
 	qdel(L)
 
 
-// This proc checks for available mothers and offers a choice if there's more than one.
+// This proc checks for available spawn points and offers a choice if there's more than one.
 /datum/hive_status/normal/proc/attempt_to_spawn_larva(mob/xeno_candidate)
```

Comment:
Do the silos currently try to get an observer via get_alien_candidate first? If yes, resolve this, if not, make it that way please.

Reply:
Silos are permanent spawn points, they are passive.
What can be done is for new burrowed larva additions to try to find observers and mothers, and then spawn.
Want that?

Reply:
1. Try to find observer and spawn them
2. If that works, return, if not, either notify observers with a popup or spawn an empty larva

That was the idea me and psy agreed on

Reply:
I'll need clarification on where you want this behavior added.
I can't add it to resin silos because they are just passive spawn points. Nor to `attempt_to_spawn_larva()` because this is merely the hive proc called by `attempt_to_join_as_larva()`.

Reply:
I don't care where it's added but any xeno spawning methods should try to use that proc to preserve the behavior that oldest observers get to play first, and only using popups and other ways in case there are none, to still offer ghosts without prefs to play.

Reply:
Noted.